backend/src/Controllers/ballots.controllers.ts

```diff
@@ -1,17 +1,16 @@
 import { Ballot as BallotType } from '../../../domain_model/Ballot';
-import { reqIdSuffix } from "../IRequest";
 import Logger from "../Services/Logging/Logger";
 import ServiceLocator from "../ServiceLocator";
 import { responseErr } from '../Util';
+import BallotsDB from '../Models/Ballots';
 
-const BallotsDB = require('../Models/Ballots');
 var BallotModel = new BallotsDB(ServiceLocator.postgres());
 const className = 'Ballots.Controllers';
 
 const ballotByID = async (req: any, res: any, next: any) => {
     try {
         const ballotId = req.params.id;
-        const ballot = await BallotModel.getElectionByID(parseInt(ballotId));
+        const ballot = await BallotModel.getBallotByID(ballotId, req);
```

Comment:
1. I set up a middleware at the top of the stack that adds the necessary Logging Context data to the request object.  There's an "IRequest" type that the request implements which also fulfills the Logging Context contract.

2. In this case, because most of the file is still using "any", the compiler has no idea what req is and just trusts us that we can use it there :P